src/geometamaker/geometamaker.py

```diff
@@ -269,7 +273,8 @@ def describe_raster(source_dataset_path, scheme):
     description['data_model'] = models.RasterSchema(
         bands=bands,
         pixel_size=info['pixel_size'],
-        raster_size=info['raster_size'])
+        raster_size={'width': info['raster_size'][0],
+                     'height': info['raster_size'][1]})
```

Comment:
I also like that this could be expanded to include the number of bands or any other attributes that expression the dimensionality of the dataset in the future, should it become necessary.